addon/utils/prop-types.js

```diff
@@ -61,6 +61,10 @@ export function generateType (key) {
         def.required = options.required
       }
 
+      if (options.updatable === false) {
+        def.updatable = false
```

Comment:
I must be missing something here. Why would we want this instead of `def.updatable = options.updatable`? Is it because if `options.updatable` is `undefined` we want `def.updatable` to be `true`?